requirements_dev.txt

```diff
@@ -2,7 +2,11 @@
 # code linting and formatting
 flake8
 flake8-docstrings
-flake8-black
+# Black 22.1 has a fatal bug when used with flake8-black and dbt, so freeze it for now.
```

Comment:
Nitpick: The comment says we're freezing versions, but we're not freezing (i.e. requesting a specific version). Rather, we're specifying minimum or maximum versions.

Reply:
Just fixed to "cap" and included a link to this issue.